contrib/etc/lockc/lockc.toml

```diff
@@ -379,6 +379,7 @@ allowed_paths_access_baseline = [
 denied_paths_access_restricted = [
     "/proc/acpi",
     "/proc/sys",
+    "/var/run/secrets/kubernetes.io",
```

Comment:
Should we add this to `baseline` as well? @flavio WDYT?

Reply:
Yes, I think that would be a good idea